__tests__/app.test.js

```diff
@@ -313,3 +313,28 @@ test("POST 400: failed to update vote as no vote sent", () => {
       expect(body.msg).toEqual("Insufficient data");
     });
 });
+describe("/api/users", () => {
+  test("GET 200: gets an array of user objects", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.users.length).toBeGreaterThan(0);
```

Comment:
As we are using the test data and we know how many users objects there are. We should test for exact length to avoid false positives